components/blitz/src/omero/cmd/mail/SendEmailRequestI.java

```diff
@@ -74,17 +74,13 @@ public SendEmailRequestI(MailUtil mailUtil) {
     public void init(Helper helper) {
         this.helper = helper;
 
-        final EventContext ec = ((LocalAdmin) helper.getServiceFactory()
```

Comment:
@aleksandra-tarkowska : I was hoping to also looking at sending emails on group creation. Could we do the same for shares? It might take some minor refactoring, but that will be much better than leaving ourselves open for abuse.